CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [Unreleased]
+## [v245] - 2024-02-08
```

Comment:
This is an incomplete change for performing a release. The links in the footer need updating too :-) Plus the `[Unreleased]` section should be left in place for the next cycle. These are all things the prepare release automation would do for you, reducing the chance of user error.

Given there are multiple changes already on `main` for this release (and so this PR isn't the only change in the release), it seems cleaner to have the "prepare release" changes be their own PR. (In the scenario where you have no changes on `main` and have a combination PR from `develop`, then I could see the argument for bundling in the release preparation into the combo PR.) 

With the new automation, the creation of the release preparation PR is very quick and can be self-reviewed, meaning there isn't another delay for a review cycle.

Lastly, should this release get delayed after this PR is merged, then with the current PR it would require a manual fixup of the release date (that would require another team member to review), whereas with the automation PR, you can re-trigger the workflow and it will update the "prepare release" PR with the new date.

Reply:
Would you mind omitting this line from the PR, and trying out the release automation at least once? :-)

Reply:
Alright, @edmorley